src/views/preview/embed-view.jsx

```diff
@@ -0,0 +1,92 @@
+// embed view
+
+const bindAll = require('lodash.bindall');
+const React = require('react');
+const PropTypes = require('prop-types');
+const injectIntl = require('react-intl').injectIntl;
+
+const ProjectViewHOC = require('./project-view-hoc.jsx');
+const ErrorBoundary = require('../../components/errorboundary/errorboundary.jsx');
+const projectShape = require('./projectshape.jsx').projectShape;
+const NotAvailable = require('../../components/not-available/not-available.jsx');
+const Meta = require('./meta.jsx');
+
+const GUI = require('scratch-gui');
+const IntlGUI = injectIntl(GUI.default);
+
+const Sentry = require('@sentry/browser');
+if (`${process.env.SENTRY_DSN}` !== '') {
+    Sentry.init({
+        dsn: `${process.env.SENTRY_DSN}`,
+        // Do not collect global onerror, only collect specifically from React error boundaries.
+        // TryCatch plugin also includes errors from setTimeouts (i.e. the VM)
+        integrations: integrations => integrations.filter(i =>
+            !(i.name === 'GlobalHandlers' || i.name === 'TryCatch'))
+    });
+    window.Sentry = Sentry; // Allow GUI access to Sentry via window
+}
+
+class EmbedView extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+        ]);
+        const pathname = window.location.pathname.toLowerCase();
+        const parts = pathname.split('/').filter(Boolean);
+        this.state = {
+            extensions: [],
+            invalidProject: parts.length === 1,
+            projectId: parts[1]
+        };
+    }
+    componentDidMount () {
+        this.props.getProjectInfo(this.state.projectId);
+        this.props.fetchProjectData(this.state.projectId);
+    }
+    render () {
+        if (this.props.projectNotAvailable || this.state.invalidProject) {
+            return (
+                <ErrorBoundary>
+                    <div className="preview">
+                        <NotAvailable />
+                    </div>
+                </ErrorBoundary>
+            );
+        }
+
+        return (
+            <React.Fragment>
+                <Meta projectInfo={this.props.projectInfo} />
+                <React.Fragment>
```

Comment:
Just noticed that you have this wrapped in a `Fragment`, but since there's only one `IntlGUI` component, you don't need that `Fragment` wrapper.

Reply:
Will do

Reply:
Done